crates/ursa-rpc-service/src/config.rs

```diff
@@ -22,6 +24,27 @@ impl Default for ServerConfig {
         Self {
             port: Self::default_port(),
             addr: Self::default_addr(),
+            origin: Default::default(),
+        }
+    }
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Clone)]
+pub struct OriginConfig {
```

Comment:
Since this only holds one type of data, it would be less code to simply add it in ServerConfig.

Reply:
Sure, although we may need many fields for different origins, not sure if they'd be nice all top level in server config instead of nested